index.md

```diff
@@ -41,7 +41,7 @@ BAS also conducted a session on astrophotography at Amrita Institute of Technolo
 
 BAS conducted an outreach program for school children at Sri Venketeshwara Rural High School on
 24th February,2024 evening at Challakere,Karnataka. Pictures and a general description will be updated soon.
-[Link to Challakere event page](/_posts/2024-02-26-Challakere-event.markdown)
+[Link to Challakere event page]({% link _posts/2024-02-26-Challakere-event.markdown %})
```

Comment:
What is this `link` for? Can't Jekyll handle simple markdown links automatically?

Reply:
I noticed this link was broken when I was testing locally - and saw [this](https://jekyllrb.com/docs/liquid/tags/#link) in jekyll documentation and was following it. Though, I see now the link is not broken in pages.bas.org.in. I dont know if this behaviour was affected due to bootstrap plugin being used (don't understand enough of it).